src/miral/x11_support.cpp

```diff
@@ -34,13 +36,42 @@ miral::X11Support::X11Support() : self{std::make_shared<Self>()}
 
 void miral::X11Support::operator()(mir::Server& server) const
 {
+    static auto const x11_displayfd_opt = "x11-displayfd";
```

Comment:
Seems arbitrary to mash display and fd into one word. Any reason to prefer that over `x11-display-fd`?

Reply:
Vague consistency with Xmir, Xwayland (which use `-displayfd`).

Reply:
Good enough for me